docker-hub/scim.md

```diff
@@ -40,6 +40,8 @@ Before making SCIM configuration changes in your IdP, navigate to [Docker Hub](h
 
 ![scim-app-provisioning](images/scim-app-provisioning.png){:width="700px"}
 
+6. Remove the unwanted attributes from your mappings and fields from the Profile editor.
```

Comment:
In the preview, this is displaying as step 1 instead of 6.

Reply:
Thank you, updated!